includes/Block/Latest_Stories_Block.php

```diff
@@ -0,0 +1,652 @@
+<?php
+/**
+ * Class Embed_Block.
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Block;
+
+use Google\Web_Stories\Embed_Base;
+use Google\Web_Stories\Tracking;
+use Google\Web_Stories\Story_Post_Type;
+use Google\Web_Stories\Media;
+
+/**
+ * Latest Stories block class.
+ */
+class Latest_Stories_Block extends Embed_Base {
+	/**
+	 * Script handle.
+	 *
+	 * @var string
+	 */
+	const SCRIPT_HANDLE = 'latest-web-stories-block';
+
+	/**
+	 * Style handle.
+	 *
+	 * @var string
+	 */
+	const STYLE_HANDLE = 'latest-stories-block-style';
+
+	/**
+	 * Block name.
+	 *
+	 * @var string
+	 */
+	const BLOCK_NAME = 'web-stories/latest-stories';
+
+	/**
+	 * Current block's block attributes.
+	 *
+	 * @var array Block Attributes.
+	 */
+	protected $block_attributes = [];
+
+	/**
+	 * Initializes the Web Stories embed block.
+	 *
+	 * @since
+	 *
+	 * @return void
+	 */
+	public function init() {
+		$this->register_script( self::SCRIPT_HANDLE, [ self::STORY_PLAYER_HANDLE, Tracking::SCRIPT_HANDLE ] );
+		$this->register_style( self::SCRIPT_HANDLE, [ self::STORY_PLAYER_HANDLE ] );
+		wp_register_script( 'amp-carousel-script', 'https://cdn.ampproject.org/v0/amp-carousel-0.1.js', [], 'v0', true );
+
+		wp_register_style(
+			self::STYLE_HANDLE,
+			WEBSTORIES_PLUGIN_DIR_URL . 'includes/assets/latest-stories.css',
+			[],
+			'v0',
```

Comment:
this should be plugin version.

Reply:
@divyarajmasani This will eventually be removed when we use the Renderer class as you can find it in the `rt/gutenberg-selected-stories` branch.

Reply:
yes, just saw it there.